src/main/java/frc/robot/util/Blinkin.java

```diff
@@ -24,25 +29,71 @@ private static Spark getController() {
    * @param tableInput numerical input for Pattern/Color to use
    */
   public static void setColor(double tableInput) {
+    isDualColor = false;
     currColor = tableInput;
   }
   /**
    * sets pattern/color for Blinkin using Blinkin enum so you don't need to reference the table for numerical ids in the documentation
    * @param color Pattern/Color to use
    */
   public static void setColor(Colors color) {
+    isDualColor = false;
     currColor = color.m_id;
   }
 
+  /**
+   * Sets colors for dual color mode. These colors will be cycled on the Blinkin every 600 milliseconds
```

Comment:
I think it's 0.5 sec now, not 0.6, comment seems to be out of date

Reply:
Okay, I've fixed this now.